02b_widgets.ipynb

```diff
@@ -769,12 +969,13 @@
    "outputs": [],
    "source": [
     "#| export\n",
-    "# %answer key/dashboard/widgets.py 70\n",
+    "# %answer key/dashboard/widgets.py 83\n",
     "\n",
     "def on_poly_order_change(change):\n",
-    "    global original_data, selected\n",
-    "    # original_data['Savitzky-Golay'] = savgol_filter(original_data['Temperature'], window_size, poly_order)\n",
-    "    selected = original_data[(original_data['Year'] >= year_range.value[0]) & (original_data['Year'] <= year_range.value[1])]"
+    "    global original_df, selected_df\n",
+    "    # catch the change in the poly_order widget value and update the original_df\n",
+    "    # original_df['Smoothed Data'] = savgol_filter(original_df['Temperature'], window_size, poly_order).round(decimal=3)\n",
```

Comment:
Also suggest shortening here to

```python
# original_df['Smoothed Data'] = savgol_filter(....
```

Reply:
Again, fixed by removing it as a comment and directing the user to edit the line.

Reply:
Note that another change needed here, which is in the solution but not the starting comment: `window_size` → `window_size.value`

Reply:
> Note that another change needed here, which is in the solution but not the starting comment: `window_size` → `window_size.value`

This was intentional, they have to change BOTH the poly_order and window_size... the approach is different depending on which we are monitoring for a 'change'.